test/runtests.jl

```diff
@@ -31,5 +31,6 @@ include("test_optimize_surface.jl")
 include("test_simulate.jl")
 include("test_Aquaculture.jl")
 include("test_Reservoir_optimization.jl")
+include("test_optimize_reservoir.jl")
```

Comment:
I called this test_optimize_reservoir.jl to mirror test_optimize_surface.jl ... Maybe it would be better to merge it with test_Reservoir_optimization.jl?